src/test/resources/features/cs/ManagementUI.feature

```diff
@@ -2,7 +2,7 @@
 Feature: ManagementUI
 
   Background:
-    Given I am logged into "CS Management UI" as user "DECS_USER"
+    Given I am logged into "CS Management UI" as user "DCU_USER"
```

Comment:
Why have we changed this user? The cs features all have DECS_User as their main user, seems a shame to mix it up